Packs/Jira/Integrations/JiraV3/JiraV3.py

```diff
@@ -1939,6 +1939,8 @@ def create_issue_command(client: JiraBaseClient, args: Dict[str, str]) -> Comman
 
     issue_fields = create_issue_fields(client=client, issue_args=args_for_api,
                                        issue_fields_mapper=client.ISSUE_FIELDS_CREATE_MAPPER)
+    if "summary" not in issue_fields.get("fields", {}):
```

Comment:
I think that you need to check if "summary" not in issue_fields.get("fields", {}) or not demisto.args()["summary"]
Am I correct?

Reply:
what @anas-yousef said.
It will always be populated in fields dict